v3/src/components/slider/slider-component.tsx

```diff
@@ -96,6 +89,11 @@ export const SliderComponent = observer(({sliderModel} : IProps) => {
     }
   }
 
```

Comment:
@kswenson will the inspector always be in a portal, since it will be a single component that will have to appear in disparate places throughout the dom and component tree?